views/playground/xfp-rules/rules.html.twig

```diff
@@ -55,424 +55,176 @@
     }}
 {%- endblock -%}
 
+{% set controls = '
+<div class="rule-block__control">
+    <div class="form__group form__group--medium">
+        <label class="control__label hide">Choose trigger</label>
```

Comment:
lots of missing `for`'s here. I know it's a playground so it doesn't matter that much but I'd add them in case someone blindly copies.

Reply:
not doing as part of this change